internal/reader/home_chain.go

```diff
@@ -348,25 +365,43 @@ type ChainConfig struct {
 	Config chainconfig.ChainConfig `json:"config"`
 }
 
-// OCR3Config mirrors CCIPConfig.sol's OCR3Config struct
+// nolint: lll
+// https://github.com/smartcontractkit/chainlink/blob/e964798a974f3246ee1da011feffe33509b358df/contracts/src/v0.8/ccip/capability/CCIPHome.sol#L105-L131
+
+type OCR3Node struct {
+	P2pID          [32]byte `json:"p2pId"`
```

Comment:
This casing is just awkward IMO. I see this in many places throughout the codebase. Better:

```suggestion
	P2PID          [32]byte `json:"p2pId"`
```